CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 8.3.2
```

Comment:
This changelog entry will need to be updated to the next hotfix version. (you will need to move it to the top of this file and change the version)